src/styles/_star.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: inline-flex;
```

Comment:
why inline-flex?

Reply:
in this moment the STAR say me that i need to use inline flex